pre-docs/plan_functions.md

```diff
@@ -1,6 +1,177 @@
 # Bolt Functions
 
 
+## :'ctrl::do_until'
```

Comment:
Oh, boo it prepends non-core functions with a `:` which I think changes the order. Perhaps we should pin yard?

Reply:
I'm fine with pinning yard if that fixes it. I also don't think it's a huge issue if it's not quick to fix. I'm more urgent about getting travis passing.